python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -673,6 +659,19 @@ def test_multicolumn_iloc(pdf, gdf, pdfIndex, iloc_rows, iloc_columns):
         assert_eq(presult, gresult, check_index_type=False)
 
 
+@pytest.mark.xfail(
+    reason="A problem with downcasting in this instance", raises=AssertionError
+)
+def test_multicolumn_item():
+    gdf = cudf.DataFrame(
+        {"x": np.arange(10), "y": np.arange(10), "z": np.arange(10)}
+    )
+    gdg = gdf.groupby(["x", "y"]).min()
+    gdgT = gdg.T
+    pdgT = gdgT.to_pandas()
+    assert_eq(gdgT[(0, 0)], pdgT[(0, 0)])
```

Comment:
Why does this test fail?

Reply:
This is a specific case of downcasting incorrectly - `gdgT[(0, 0)]` returns a `DataFrame`, `pdgT[(0, 0)]` returns a `Series`. Fixing this specific downcast issue is slightly harder than trivial and isn't in the same scope as this `__repr__` fix, so I'm pushing it. https://github.com/rapidsai/cudf/issues/2711 catalogues a couple of other MultiIndex downcasting/shape issues. I will add this to it.

Reply:
This is fixed in the latest.